frontend/pagesetup.css

```diff
@@ -0,0 +1,18 @@
+html, body { font-family: "Oxygen Mono";
+       background: #f0f0f0;
+       background: linear-gradient(to bottom, rgba(225,232,252,1) 0%, rgba(242,242,242,1) 100%); }
+
+section { width: 100%; }
+
+h1 > a > img { height: 1em; }
+
+
+@font-face {
+  font-family: 'Oxygen Mono';
+  src: local('Oxygen Mono'), url('oxygen-mono-regular.woff'), url('oxygen-mono-regular.ttf');
+}
+
+@media (min-width: 700px) {
+  section { width: 60%; padding: 42px 20%; min-height: 42vh; }
+  img { max-width: 700px; }
+}
```

Comment:
I'd suggest to use consistent indentation style in the future, but it's not really important, so feel free to ignore my nagging :)